src/appleseed.shaders/src/appleseed/as_standard_surface.osl

```diff
@@ -97,6 +97,20 @@ shader as_standard_surface
         int as_maya_attribute_hidden = 1,
         int gafferNoduleLayoutVisible = 0
     ]],
+    int in_sss_profile = 0
+    [[
+        string as_maya_attribute_name = "sssProfile",
+        string as_maya_attribute_short_name = "ssp",
+        string widget = "mapper",
+        string options = "Better Dipole:0|Directional Dipole:1|Gaussian:2|Normalized Diffusion:3|Standard Dipole:4|Random Walk:5",
```

Comment:
In terms of profiles, in my opinion all plugins should only expose (in this order):

- "Gaussian"
- Better Dipole with the name "Dipole"
- "Normalized Diffusion"
- "Random Walk"